### ISSUE TITLE
Calling score_test_set multiple times leads to Runtime Error

### ISSUE THREAD

**Issue by DreRnc** [plinder-org/plinder#88]
Hi, because in the code for the scoring function the context is set, multiple calls to this function will lead to a Runtime Error "context has already been set".

https://github.com/plinder-org/plinder/blob/9658cc294722c164f15241f478c4e612cfa437d8/src/plinder/eval/docking/write_scores.py#L215-L231

I think it isn't best practice to set the context inside a function call that could be theoretically called multiple times. Or at least, it should be checked that the context is not already set before trying to set it.

As an example of the use case problem, I encountered this when scoring inference for multiple models. So because of this, I need to call as many processes as models I have to score.

Fixing this should be extremely quick (an if statement to check that the context is not set), I can open a pull request for it.

**Comment by Ninjani** [comment]
Thanks for the catch, please open a PR! I guess this should also work: `multiprocessing.get_context("spawn").Pool(num_processes)`?

**Comment by DreRnc** [comment]
I agree with your solution, starting a local context is better if you want it always to use "spawn". I will open a PR now and merge it into contrib_dev.
